src/plugin-main.cpp

```diff
@@ -188,19 +188,21 @@ bool is_module_found(const char *module_name)
 		[](void *param, const struct obs_module_info2 *module_info) {
 			struct find_module_data *data_ =
 				(struct find_module_data *)param;
-			if (strcmp(data_->target_name, module_info->name) ==
+			if (stricmp(data_->target_name, module_info->name) ==
```

Comment:
Oh, I didn't really mean for you to use my code; I liked your code better.
TIL that ANSI C does not define `stricmp`!
Supposedly there is `strcasecmp`, but supposedly that is also not guaranteed.
Maybe just revert back to `strcmp`?